src/app/userDashboard/registerMatches/page.tsx

```diff
@@ -76,50 +67,49 @@ const regMatches = () => {
               </h1>
             </div>
             <div className={styles.sendmailbtnContainer}>
-              {/* <button
-                  className={styles.sendMailBtn}
-                  onClick={() => setPoolModal(true)}>
-                  
-                  SEND INVITE BY EMAIL
-                </button> */}
             </div>
           </div>
           <div className={styles.room_wrapper}>
             <div className={styles.room_container}>
               <div className={styles.registeredmatches}>
                 <div className={styles.imgSection}>
                   <Image
-                    src="../assests/userdashboardbg.svg"
+                    src={matchData?.mapImg || "../assests/userdashboardbg.svg"}
                     alt="userdashboardbg"
                     className={styles.wrapperimg}
-                    width={100}
-                    height={100}
+                    width={200}
+                    height={200}
                   />
                 </div>
-                {/* <span className={styles.register_match}>
-                  Registered Matches
-                </span> */}
               </div>
               <div className={styles.squad_match}>
-                <span className={styles.register_match}>
+                <span className={styles.register_match_gamename}>
                   {matchData?.gameName}
                 </span>
                 <span className={styles.winning_prize}>
-                  Time : {matchData?.date} at {matchData?.time}
+                  Time: {matchData?.dateAndTime.toString()}
                 </span>
 
                 <div className={styles.winnings}>
                   <div>
                     <span className={styles.winning_prize}>WINNING PRIZE</span>
```

Comment:
Have you checked in userTpurnament screen for same also if you understand react component this whole section is a reusable component